### PR TITLE
new-branch

### PR DESCRIPTION
**Pull request by ElShaer398**
Hello_world_project

### CODE REVIEW

**Review by frank1738 — COMMENTED** [review]
Hi @ElShaer398 , 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.



## Invalid Code Review Request

You have submitted a project, that has linter errors. This is probably because your `css` file is empty. Kindly add some styling. I would also recommend that you use the [Microverse readme template](https://github.com/microverseinc/readme-template) and customize it so that your documentation looks professional;

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

File: test.md [hunk 1]
```diff
@@ -0,0 +1 @@
+This file should be ignored by git
```
Comment (mckent05) [inline test.md:1]: -According to the project requirements, this file should be ignored by git, but I can still view it in your project. Please try to fix this 👍 👍

**Review by mckent05 — CHANGES_REQUESTED** [review]
Hi @ElShaer398,

# STATUS: CHANGES REQUIRED ♻️ ♻️  

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
almost there

## Highlights
- GitHub flow correctly followed 👍 
- You have done a good job setting up your project 👏 

## Required Changes ♻️
- To make your repo look more professional please consider the following points:
-- It is important to name your branches correctly, the name of your branch could be the name of a feature or action. In this instance, a suitable name could be `feature_branch` or `Hello_world_branch`. Please take note of this in your subsequent project 🙏 
-- When making a PR it is important that your PR summary and title and descriptive, The PR summary should briefly summarise the changes you have made in that PR, and your PR title should be as descriptive as possible, in this case a suitable name could be `Hello_world_project`.
-- You can edit your PR title by clicking on the three dots as referenced in the picture below, please edit your PR summary and include a more descriptive summary.
pr edit
 
_Check the comments under the review._

## Optional suggestions
Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: test.md (hunk 1 above)
Comment (ElShaer398) [reply test.md:1]: image
I don't know why is it still visible.

File: README.md [hunk 2]
```diff
@@ -1 +1,15 @@
...
+GitHub: https://github.com/ElShaer398
+LinkedIn: https://www.linkedin.com/in/mohammad-elshaer-29846b195/
+
+Feel free to check the issues page.
+
+Show your support
+Give a ⭐️ if you like this project!
+
+📝 License
+This project is MIT licensed.
```
Comment (HAMMAS-SALEEM) [inline README.md]: **[Optional]**
- [ ] Kindly make your pull request look better to make it easily readable. Now what you can see are words. But if would make it more stylish then this file would be easily readable 👍 You can use this [template code](https://raw.githubusercontent.com/microverseinc/readme-template/master/README.md) to make it stylish

**Review by HAMMAS-SALEEM — APPROVED** [review]
# Status: Approved :green_circle: :1st_place_medal: :100:
success

## Hey @ElShaer398 :wave:

Great work on implementing the requested changes. Your project is complete. :blush:
There is nothing else to say about your project. :+1:

## Highlights :1st_place_medal:

:heavy_check_mark: Descriptive PR Title
:heavy_check_mark: Descriptive PR Summary
:heavy_check_mark: Following Github flow
:heavy_check_mark: Following Requirements

# Optional

_All the changes with optional tags are not crucial enough to prevent you from getting the approval but we highly recommend to students that they should take these changes into consideration._

Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

---------------------------------------

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**